### PR TITLE
Safari scroll issue

### LINKED ISSUE #862: Scroll doesn't work using Safari
### Expected behaviour
Scrolling down the page should work in safari

### Actual behaviour
Scrolling doesn't work in safari

### Steps to reproduce the behaviour

- Use Safari browser
- Select UKIS 1_0001.json schema
- Try scrolling down the page

#### Browser
Safari 10.0.1

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Fixes #862 Scroll issue in safari 

### How to review 

- Use Safari
- Select UKIS 1_0001.json schema
- Try scrolling down the page

### CODE REVIEW

File: app/assets/styles/partials/objects/_page.scss [hunk 1]
```diff
@@ -47,8 +48,10 @@
 
 .page__content {
   flex: 1 0 auto;
+  padding-bottom: 1px;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_page.scss:53]: Can we at least add a comment in the code with a link to this bug so we know why this seemingly arbitrary bit of padding his here?

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Review by hamishtaplin — APPROVED** [review]

**Comment by ajmaddaford** [comment]
@bennpearson I don't get this issue on master, I'm using Safari on a Mac
image

**Comment by hamishtaplin** [comment]
@ajmaddaford It's a definite issue, I can reproduce—it happens on the introduction page.

**Comment by ajmaddaford** [comment]
@hamishtaplin I believe it's an issue, but I couldn't test that this fixes it! I do get an additional scroll bar without this fix, and it fixes that.

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**